tests/test_stamp_parity.py

```diff
@@ -8,73 +8,44 @@
 
 import numpy as np
 
-from kbmod.fake_data_creator import add_fake_object
+from kbmod.fake_data_creator import FakeDataSet
 from kbmod.search import *
+from kbmod.trajectory_utils import make_trajectory
 
 
 class test_search(unittest.TestCase):
     def setUp(self):
-        # test pass thresholds
-        self.pixel_error = 0
-        self.velocity_error = 0.05
-        self.flux_error = 0.15
-
         # image properties
-        self.imCount = 20
+        self.img_count = 20
         self.dim_x = 80
         self.dim_y = 60
-        self.noise_level = 4.0
```

Comment:
I'm assuming that many of these are parameters that are no longer needed.

Reply:
Correct